src/renderer/components/atoms/text/text.tsx

```diff
@@ -0,0 +1,16 @@
+import type { TextProps } from './types';
+import { resolveElement, buildStylesFor, returnTextColor } from './helper';
+
+const Text = ({ as, variant, useColor, className, children }: TextProps) => {
```

Comment:
can we rename the `useColor` prop to just `color`? `useColor` feels like a hook

Reply:
Agree, done!